src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js

```diff
@@ -0,0 +1,58 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+
+import claimsList from './fixtures/mocks/claims-list.json';
+
+describe('Breadcrumb Test', () => {
+  it('Verifies breadcrumb functionality', () => {
+    cy.intercept('GET', '/v0/evss_claims_async', claimsList);
+    cy.visit('/track-claims');
+    cy.login();
```

Comment:
Not sure if it matters here, but `cy.login()` usually comes before `cy.visit()`.

Reply:
Checked and it works the same both ways, but went ahead and moved it in each test to before as if that's a standard, I don't want to be the one that undoes our norm!  Moved it in all the tests and they all still pass as they did before.